pkgs/tools/networking/slurm-cli/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, meson
+, ncurses
+, ninja
+}:
+
+stdenv.mkDerivation rec {
+  pname = "slurm";
+  version = "0.4.4";
+
+  src = fetchFromGitHub {
+    owner = "mattthias";
+    repo = "slurm";
+    rev = "v${version}";
+    hash = "sha256-w77SIXFctMwwNw9cQm0HQaEaMs/5NXQjn1LpvkpCCB8=";
+  };
+
+  nativeBuildInputs = [ pkg-config meson ninja ];
+  buildInputs = [ ncurses ];
+
+  intallPhase = ''
+    mkdir -p "$out/bin"
+  '';
```

Comment:
Normally I'd say, please keep the phase hooks as mentioned here when overriding a phase: https://nix.dev/tutorials/learning-journey/packaging-existing-software#phases-and-hooks

But this code is not doing anything, best remove it.

```suggestion
```